source/includes/mongodb-compatibility-table-java.rst

```diff
@@ -208,7 +208,9 @@
      -
      - |checkmark|
 
-.. [#driver-support] The 2.14 driver does not support all MongoDB 3.2 features (e.g.,
+.. [#3.12-driver-support]  The 3.12 driver does not support all MongoDB 4.4 features
+
+.. [#2.14-driver-support] The 2.14 driver does not support all MongoDB 3.2 features (e.g.,
```

Comment:
Nitpick: I don't think we should add a comma directly after "e.g.". We should probably also avoid "e.g." and parentheses (I think the style guide mentions both of these) and instead:
"... all MongoDB 3.2 features such as read concern. However, ..."